lib/icingadb/icingadb.ti

```diff
@@ -10,7 +10,7 @@ namespace icinga
 
 class IcingaDB : ConfigObject
 {
-	activation_priority 100;
+	activation_priority -50;
```

Comment:
> not yet activated objects – all checkables – will effectively be deleted

Idk, how you came up to that conclusion, nor how this relates to #10057 or #10151 in anyway. Icinga DB won't do anything if the checkables aren't **truly** deleted.
https://github.com/Icinga/icinga2/blob/321890859540a37b8b7f72fe74539094cfe9c867/lib/icingadb/icingadb-objects.cpp#L2781-L2782

I already checked this PR last week and noticed that Icinga DB, IDO etc. are not activated first as before, but _last_ in the same way they are deactivated. IMHO the possible solution I am thinking of is to not touch the activation order of these objects on startup and deactivate them last on shutdown/reload, but this will not be achievable by simply changing the `activation_priority` field. So I wanted to wait until tomorrow and discuss it in our next meeting.